salt/modules/portage_config.py

```diff
@@ -205,13 +205,26 @@ def _package_conf_ordering(conf, clean=True, keep_backup=False):
                         triplet[0] != path:
                     shutil.rmtree(triplet[0])
 
-
-def _merge_flags(*args):
+def check_accept_keywords(approved, flag):
```

Comment:
This function needs a docstring.  If it is intended to be externally available, `portage_config.check_accept_keywords`, then it also needs a `.. versionadded:: 2014.7.5`.